pages/facts.html

```diff
@@ -2,11 +2,30 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>National Parks Fun Facts</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/facts.css".css>
 </head>
 <body>
-  
+    <header class="top-bar">
+      <h1>Fun Facts</h1>
+      <nav id="navlinks">
+        <ul>
+          <li><a href="../index.html">Home</a></li>
+          <li>Facts</li>
+          <li><a href="gallery.html">Gallery</a></li>
+          <li><a href="../pages/blog/leave-no-trace.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <div class="facts">
```

Comment:
This container has flex applied, but has only one child (the `ul`). While this _can_ be helpful, here the change is so minor, I'd probaby leave it off.